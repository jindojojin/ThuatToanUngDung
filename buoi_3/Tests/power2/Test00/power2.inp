20
1 4 5 1 4 5 3 3 3 3 7 3 4 5 3 4 5 64 32 3
