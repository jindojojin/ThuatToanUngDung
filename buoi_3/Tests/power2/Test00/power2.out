1 1 4 4 4 4 32 64
